### ISSUE TITLE
Do you have a wordpress version to buy

### ISSUE THREAD

**Issue by rifatozkancomtr** [wowthemesnet/template-pintereso-bootstrap-jekyll#1]
Do you have a wordpress version to buy

**Comment by codebygina** [comment]
No, I didn't know anyone was interested. Do you think more people would be interested?

**Comment by rifatozkancomtr** [comment]
https://themeforest.net/item/pinthis-pinterest-style-wordpress-theme/7259295?s_rank=1


you can see the sales quantity of similar themes.

**Comment by codebygina** [comment]
Oh I see. I will release a WP version then soon. Thanks!

**Comment by codebygina** [comment]
If anyone interested, the WordPress version is out: https://www.wowthemes.net/themes/pinpresso-wordpress/
